### PR TITLE
Add iOS delegate documentation

### PR DESCRIPTION
**Pull request by diegoserranoa**
Add iOS delegate documentation (callbacks)

## Trello
https://trello.com/c/dZM7Wz45/4610-3-ios-sdk-callbacks-25

### CODE REVIEW

File: source/includes/_ios_delegate.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Survey Delegate
+
+The `WTRSurveyDelegate` can be used to be informed of when the SDK will or
+already perform certain actions. To use the delegate, first a class needs
+to conform to the protocol and assign that class to the WootricSDK.
+
+Imagine you have a view controller `SampleViewController.m`. You can
+specific that this class conforms to `WTRSurveyDelegate` and it will be
```
Comment (hachibu) [inline source/includes/_ios_delegate.md]:
```suggestion
specify that this class conforms to `WTRSurveyDelegate` and it will be
```

Comment (hachibu) [inline source/includes/_ios_delegate.md]: Maybe there's a better way to say this, like maybe use the word inheritance or inherit.

File: source/includes/_ios_delegate.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+}
+```
+
+## willPresentSurvey
+
+Called by the WootricSDK when the survey is about to appear.
+
+## didPresentSurvey
+
+Called by the WootricSDK when the survey already appeared.
```
Comment (hachibu) [inline source/includes/_ios_delegate.md]:
```suggestion
Called by the WootricSDK when the survey has already appeared.
```

File: source/includes/_ios_delegate.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## willHideSurvey
+
+Called by the WootricSDK when the survey is about to be dismissed.
+
+## didHideSurvey
+
+Called by the WootricSDK when the survey already disappeared. This method
```
Comment (hachibu) [inline source/includes/_ios_delegate.md]:
```suggestion
Called by the WootricSDK when the survey has disappeared. This method
```

**Comment by diegoserranoa** [comment]
@hachibu thanks for the comments! Fixed the language ✅

**Review by hachibu — APPROVED** [review]
🚢

**Pull request merged**